std_msgs/Header header
int32 left_position
int32 left_speed
int32 left_force
int32 right_position
int32 right_speed
int32 right_force